Tests/SonarRunner.Shim.Tests/SonarRunnerWrapperTests.cs

```diff
@@ -20,7 +19,7 @@ public class SonarRunnerWrapperTests
         #region Tests
 
         [TestMethod]
-        public void SonarRunnerHome_NoWarning()
+        public void SonarRunnerHome_NoMessageIfNotAlreadySet()
```

Comment:
Miscellaneous test changes, unrelated to bug but necessary to get the tests to pass on a machine that does have SONAR_RUNNER_HOME set.

I renamed the tests - they produce diagnostic messages rather than warnings.